crates/subspace-farmer/src/farming.rs

```diff
@@ -45,12 +52,14 @@ impl Farming {
             receiver,
         ));
 
+        // return the instance
```

Comment:
This comment is honestly quite useless, you don't need to write something that literally repeats the code